src/Check/PhpFlag.php

```diff
@@ -24,44 +18,27 @@
  */
 class PhpFlag extends AbstractCheck implements CheckInterface
 {
-    /** @var array */
+    /** @var non-empty-list<string> */
     protected $settings;
 
     /** @var bool */
     protected $expectedValue;
 
     /**
-     * @param string|array|Traversable $settingName   PHP setting names to check.
-     * @param bool                     $expectedValue true or false
+     * @param string|non-empty-list<string> $settingName   PHP setting names to check.
```

Comment:
Similar query RE Traversable satisfying this

Reply:
`if (count($this->settings) > 1) {` used below :thinking: